data/projects.json

```
@@ -368,5 +368,15 @@
         "level": "intermediate",
         "site": "https://barber-shop-53333.web.app",
         "tags": "vuejs, firebase, html, js, css, barber-shop"
+    },
+    {
+        "name": "Digital Farkle",
+        "display": "Farkle",
+        "description": "Farkle is a dice game very similar to Yahtzee, but not as complex. Simple UI where players roll dice to hit specified point values",
+        "author": "MonkeyToji",
+        "category": "game",
+        "level": "intermediate",
+        "site": "https://github.com/MonkeyToji/DigitalFarkle",
+        "tags": "python, game"
```

Please add a tag for unique key and it will be linked with `ideas.json`
```suggestion
        "tags": "farkle, python, game"
```